src/sentry/apidocs/api_ownership_stats_dont_modify.json

```diff
@@ -671,6 +671,7 @@
             "OrganizationProjectsExperimentEndpoint::POST",
             "OrganizationSCIMMemberDetails::PUT",
             "OrganizationSCIMTeamDetails::PUT",
+            "ReleaseThresholdIndexEndpoint::GET",
```

Comment:
Is this file edited automatically?

Reply:
yeah its automatically generated i believe @sentaur-athena

Reply:
Yes, please ignore the changes here.